libr/core/cio.c

```diff
@@ -292,24 +292,12 @@ R_API bool r_core_seek(RCore *core, ut64 addr, bool rb) {
 	}
 	if (rb) {
 		ret = r_core_block_read (core);
-		if (core->io->ff) {
-			if (ret < 1 || ret > core->blocksize) {
-				memset (core->block, core->io->Oxff, core->blocksize);
-			} else {
-				memset (core->block+ret, core->io->Oxff, core->blocksize-ret);
```

Comment:
You committed into master a fix to put space into core->block + ret and in siol this is removed what means you fucked up my branch with something I don't know yet whether is right or not or even with more discussion since in siol io->0xff is gone.

```
-                               memset (core->block+ret, core->io->Oxff, core->blocksize-ret);
+                               memset (core->block + ret, core->io->Oxff, core->blocksize - ret);
---
-                       if (ret < 1)
+                       if (ret < 1) {
                                core->offset = old;
+                       }
```

What I expect is to take the diff verbatim from siol to master when is not relevant with io and should have been fixed in master if that is possible ofc, thus this is very subtle. So I don't know what you did that commit when is something does not come from siol.

It would be great for the next time to know from which commit you are taking the diff that way I might split a commit in more, dropping the change your moved into master.

cc @radare